test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -1829,3 +1842,88 @@ UCS_TEST_SKIP_COND_P(test_ucp_address_v2, diff_seg_sizes,
 }
 
 UCP_INSTANTIATE_TEST_CASE(test_ucp_address_v2)
+
+class test_pci_bw : public test_ucp_wireup {
```

Comment:
maybe this test can be in test_ucp_context.cc or test_ucp_worker.cc?
sees it does not need endpoints and not related to wireup